### PR TITLE
Portfolio: validate contact form

### PR DESCRIPTION
**Pull request by mohamedelshamy55**
Process user input according to business rules.
Use client-side validation to catch and throw errors in the UI.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -8,6 +8,13 @@ import {
 
 import modelHandler from './popup.js';
 
+import {
+  inputs,
```
Comment (carlos-ssh) [inline main.js:12]: **Kindly work on this requested changes:**
- Add a function to validate the email input that can check your input text, if it's typed correctly in lowercase.

File: main.js [hunk 2]
```diff
@@ -8,6 +8,13 @@ import {
 
 import modelHandler from './popup.js';
```
Comment (carlos-ssh) [inline main.js:10]: # 💾 💾 💽 [OPTIONAL]   💽 💾 💾
**✔️ _Keep professionalism in your Repo's_** 

You can improve your repositories management:
- Create a assets directory and save all your content there.

Screen Shot 2021-11-10 at 17 33 08

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @mohamedelshamy55 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._


changesrequested




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
### Hi @mohamedelshamy55  🙋‍♀️, 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**